A31-xds-routeaction-support.md

```diff
@@ -157,42 +157,42 @@ red box represents the top-level Load Balancing Policy and its children.
 
 #### Supported Fields
 
-Initially, only timeout-related `RouteAction` settings will be supported:
-[`timeout`](https://github.com/envoyproxy/envoy/blob/7abb0e0bbed4f6b6304403b93762614ad385f80d/api/envoy/api/v2/route/route_components.proto#L886)
+Initially, only the timeout-related `RouteAction` settings of
+[`max_stream_duration.max_stream_duration`](https://github.com/envoyproxy/envoy/blob/6f2ad057f8655e2688a195337f7520dbd77015fa/api/envoy/config/route/v3/route_components.proto#L775)
```

Comment:
We should also document that we will support the [max_stream_duration](https://github.com/envoyproxy/envoy/blob/6f2ad057f8655e2688a195337f7520dbd77015fa/api/envoy/config/core/v3/protocol.proto#L95) field in the [HttpConnectionManager.common_http_options](https://github.com/envoyproxy/envoy/blob/6f2ad057f8655e2688a195337f7520dbd77015fa/api/envoy/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto#L289) field as a default if the per-route value is unset.

Reply:
Done